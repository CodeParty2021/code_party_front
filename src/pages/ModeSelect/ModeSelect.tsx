import React from "react";
import { Link } from "react-router-dom";

type Props = {};

export const ModeSelect: React.FC<Props> = () => {
  return (
    <div>
      <div>遊ぶモードを決める</div>
      <Link to="/robot-development/top">ロボット開発モード</Link>
      <Link to="/casual-battle">カジュアル対戦モード</Link>
    </div>
  );
};
