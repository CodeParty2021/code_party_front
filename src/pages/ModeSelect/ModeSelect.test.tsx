import React from "react";
import { shallow } from "enzyme";
import { ModeSelect } from "./ModeSelect";

describe("<ModeSelect />", () => {
  it("auth snapshot test", () => {
    const wrapper = shallow(<ModeSelect />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
