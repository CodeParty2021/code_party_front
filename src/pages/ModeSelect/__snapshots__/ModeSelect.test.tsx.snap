// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModeSelect /> auth snapshot test 1`] = `
Array [
  <div>
    <div>
      遊ぶモードを決める
    </div>
    <Link
      to="/robot-development/top"
    >
      ロボット開発モード
    </Link>
    <Link
      to="/casual-battle"
    >
      カジュアル対戦モード
    </Link>
  </div>,
]
`;
